data-esnext.js

```diff
@@ -51,7 +51,7 @@ exports.tests = [
   spec: 'http://wiki.ecmascript.org/doku.php?id=strawman:do_expressions',
   exec: function () {/*
     return do {
-      let x = 23;
+      var x = 23;
```

Comment:
do expressions aren't specified, but once they are, it's not even clear they'll support var declarations inside of them.